### PR TITLE
Assignment4

### PR DESCRIPTION
**Pull request by ChrisPaneque**

### CODE REVIEW

File: Assignment4/c.paneque-21A280/task06.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+#Automatically generated by Colaboratory.
+
+#Original file is located at
+   # https://colab.research.google.com/drive/1TKFH51IowiMiu6TQLp-WiduCnFnxRKkk
+
+#**Task 06: Modifying RDF(s)**
+#"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/c.paneque-21A280/task06.py:12]: This won't run in Python. You need to comment out line 12. The same applies for the other tasks

File: Assignment4/pablohdez98-21A289/task08.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+"""Tarea: lista todos los elementos de la clase Person en el primer grafo (data01.rdf) y completa los campos (given name, family name y email) que puedan faltar con los datos del segundo grafo (data02.rdf). Puedes usar consultas SPARQL o iterar el grafo, o ambas cosas."""
+
+from rdflib.namespace import RDF, RDFS
+
+NS = Namespace("http://data.org#")
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+
+for s1, p1, o1 in g1.triples((None, RDF.type, NS.Person)):
+  for s2, p2, o2 in g2.triples((s1, None, None)):
+    g1.add((s1, p2, o2))
```
Comment (dgarijo) [inline Assignment4/pablohdez98-21A289/task08.py:30]: This is completing all persons, not just the fields requested by the task.

**Comment by dgarijo** [comment]
@ChrisPaneque, please remove the files from this PR. Otherwise I will close it and leave it to you to do another one.

**Comment by ChrisPaneque** [comment]
Hi professor, I still don't get the issue, there is just 1 Assignment 4 
in my repo.

**Comment by dgarijo** [comment]
the last update seems to have fixed it

**Pull request merged**